cmd/panasas/config/client.go

```diff
@@ -205,7 +207,7 @@ func (c *Client) GetObject(objectName string) (dataReader io.ReadCloser, oi *Obj
 	}
 	if contentType != expectedContentType {
 		err = ErrUnexpectedContentType(contentType)
-		return
+		return nil, nil, err
```

Comment:
Actually, this return should work as expected and returns `nil, nil, err` as defined in function signature but both variants should work